source/blog/2018-03-06-update-kernel-arguments-on-atomic-host.html.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Update Kernel arguments on Atomic Host
+author: rubao
+date: 2018-03-06 17:00:00 UTC
+published: true
+comments: false
+tags:
+- atomic
+- rpm-ostree
+---
+
+Users or adminstrators may want to change [kernel arguments](https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html) of Atomic Host for various reasons.
+Before, it seemed hard for the users to do so due to many of
+the steps involved, and the harmful consequeces that can occur if users accidentally
+make a mistake in the changing process.
+
+In this post, I want to introduce a command(rpm-ostree ex kargs) that
+allows users to change kernel arguments on Atomic Host. This command simplifies
+the process of changing kernel arguments into few simple commands. This command also
+currently lies beneath [rpm-ostree](https://github.com/projectatomic/rpm-ostree),
+and because of that, it benefits from many of the cool features from rpm-ostree.
+One of them is `rpm-ostree rollback`, which can allow users to undo their old changes
+they do not want.
+
+Note: This command is still under experimentation, so if you have seen any
+unexpected behavior happening, please report an issue to rpm-ostree. This
+post also might be a bit difficult to understand without prior knowledge to
```

Comment:
Recommendation: "This post requires some knowledge of Atomic Host and rpm-ostree."